doc/api/async_hooks.md

```diff
@@ -2,7 +2,10 @@
 
 <!--introduced_in=v8.1.0-->
 
-> Stability: 1 - Experimental
+> Stability: 1 - Experimental. We plan to remove this API
```

Comment:
I think this will break the stability banner display.

Reply:
It breaks the doc build entirely, actually.

<img width="878" alt="image" src="https://user-images.githubusercontent.com/718899/200186853-1a0dc8eb-1baa-488d-b863-f97d8b38c023.png">

Reply:
As @aduh95 pointed out, we’re using this formatting for `fetch`, where it seems to work: https://nodejs.org/api/globals.html#fetch

Reply:
I’m happy to change to whatever formatting you recommend.

Reply:
Oh, whoa, OK, that's weird. Let me look a bit closer at this....

Reply:
Our tooling doesn't let you add stuff to the stability banner if you're at the top level. I'll see if I can get a fix in.

Reply:
So it looks like `stability.mjs` (which builds the stability table) breaks if there are line returns in the stability index.

Reply:
I haven't looked into why this is the case (and...uh...I might not) but if you move the stability stuff below the comment that it is currently above, it will work.

Reply:
I moved it and force-pushed.

Reply:
Argh, that made the build passed, but broke things in that the API is now not marked as experimental in the table that gets generated in the main page, etc.

Reply:
https://github.com/nodejs/node/pull/45346

Reply:
> #45346

Let me test that but I don't *think* that's the right fix.

Reply:
Nope, I'm wrong, that works fine too.